src/django/pfb_network_connectivity/urls.py

```diff
@@ -29,6 +29,9 @@
 router.register(r'users', user_views.PFBUserViewSet, base_name='users')
 router.register(r'analysis_jobs', analysis_views.AnalysisJobViewSet, base_name='analysis_jobs')
 router.register(r'neighborhoods', analysis_views.NeighborhoodViewSet, base_name='neighborhoods')
+router.register(r'neighborhood_geojson',
```

Comment:
Why singular `neighborhood`?  Plural like the other endpoint would seem more regular to me.